config/config-api/src/com/thoughtworks/go/util/command/EnvironmentVariableContext.java

```diff
@@ -227,25 +224,21 @@ public int hashCode() {
         return properties != null ? properties.hashCode() : 0;
     }
 
-    public String reportText() {
-        throw new RuntimeException("IMPLEMENT ME");
-    }
-
-    public void setupRuntimeEnvironment(Map<String, String> env, ConsoleOutputStreamConsumer consumer) {
+    public List<String> report() {
```

Comment:
We can actually have a discussion about whether `processBuilder.environment()` should even be considered for this overriding behavior. But, it used to be, before this change.